app/controllers/users_controller.rb

```diff
@@ -1,5 +1,26 @@
 class UsersController < ApplicationController
+  attr_reader :user
+
   def new
     @user = User.new
   end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
find thì nó chỉ tìm theo id thôi, truyền vào 1 số thì where id = , nếu truyền nhiều số thì where id in []
find_by thì tìm theo gì cũng đc, 2 tham số, tham số 1 là cái mình muốn dùng để tìm, tham số 2 là cái để where tham số 1 =